Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -19,6 +19,8 @@ private let CancelString = NSLocalizedString("Cancel", comment: "Cancel button")
 private let KVOLoading = "loading"
 private let KVOEstimatedProgress = "estimatedProgress"
 
+private let maxDialogUrlLength = 225
```

Comment:
Sorry, it's a Vim/sed command. It means "substitute" "URL" for "Url" in this line. We often use this as shorthand for "change this".

so:

```
private let maxDialogURLLength = 225
```